### PR TITLE
Update Sass Build Process to use Webpack and update USWDS to 3.0

### PR DESCRIPTION
**Pull request by JackRyan1989**
[Ticket](https://github.com/usdoj-crt/beta.ADA.gov-Project-Management/issues/499)
[Ticket](https://github.com/usdoj-crt/beta.ADA.gov-Project-Management/issues/496)

## What does this change?

The USWDS has been updated to 3.0
We are now using Webpack to build our styles. 

## Checklist:

Confirm that styling is identical to live site on the following pages:

### Desktop resolution (1024px +):

+ [x] /notices/2021/08/25/covid-qa/ 
+ [x] /resources/ai-guidance/ 
+ [x] /resources/disability-rights-guide/
+ [x] /resources/effective-communication/
+ [x] /resources/restriping-parking-spaces/
+ [x] /resources/service-animals-faqs/
+ [x] /resources/service-animals-2010-requirements/
+ [x] /resources/web-guidance/
+ [x] /topics/intro-to-ada/
+ [x] /topics/service-animals/
+ [x] /topics/parking/
+ [x] /topics/effective-communication/
+ [x] /topics/mobility-devices/
+ [x] /topics/parental-rights/
+ [x] /topics/title-ii/
+ [x] /topics/title-iii/
+ [x] /law-and-regs/ada/
+ [x] /law-and-regs/cases/
+ [x] /feedback/
+ [x] /file-a-complaint/
+ [x] / 
+ [x] /law-and-regs/
+ [x] /infoline/
+ [x] /es/presente-una-queja/
+ [x] /resources/ 

### Tablet resolution (640 px  - 1023 px):

+ [x] /notices/2021/08/25/covid-qa/ 
+ [x] /resources/ai-guidance/ 
+ [x] /resources/disability-rights-guide/
+ [x] /resources/effective-communication/
+ [x] /resources/restriping-parking-spaces/
+ [x] /resources/service-animals-faqs/
+ [x] /resources/service-animals-2010-requirements/
+ [x] /resources/web-guidance/
+ [x] /topics/intro-to-ada/
+ [x] /topics/service-animals/
+ [x] /topics/parking/
+ [x] /topics/effective-communication/
+ [x] /topics/mobility-devices/
+ [x] /topics/parental-rights/
+ [x] /topics/title-ii/
+ [x] /topics/title-iii/
+ [x] /law-and-regs/ada/
+ [x] /law-and-regs/cases/
+ [x] /feedback/
+ [x] /file-a-complaint/
+ [x] / 
+ [x] /law-and-regs/
+ [x] /infoline/
+ [x] /es/presente-una-queja/
+ [x] /resources/ 

### Mobile resolution (Up to 639 px):

+ [x] /notices/2021/08/25/covid-qa/ 
+ [x] /resources/ai-guidance/ 
+ [x] /resources/disability-rights-guide/
+ [x] /resources/effective-communication/
+ [x] /resources/restriping-parking-spaces/
+ [x] /resources/service-animals-faqs/
+ [x] /resources/service-animals-2010-requirements/
+ [x] /resources/web-guidance/
+ [x] /topics/intro-to-ada/
+ [x] /topics/service-animals/
+ [x] /topics/parking/
+ [x] /topics/effective-communication/
+ [x] /topics/mobility-devices/
+ [x] /topics/parental-rights/
+ [x] /topics/title-ii/
+ [x] /topics/title-iii/
+ [x] /law-and-regs/ada/
+ [x] /law-and-regs/cases/
+ [x] /feedback/
+ [x] /file-a-complaint/
+ [x] / 
+ [x] /law-and-regs/
+ [x] /infoline/
+ [x] /es/presente-una-queja/
+ [x] /resources/ 

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.

### CODE REVIEW

**Comment by JackRyan1989** [comment]
@billyhunt @c2nelson I went through and tested all of the pages today. There were a few changes that needed to be made, but overall it looks good! 

Can both of you just do a quick visual inspection of each page and let me know if you have any questions before approving?

File: _assets/js/dist/accordion-compiled.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
+/*
```
Comment (billyhunt) [inline _assets/js/dist/accordion-compiled.js:1]: You might consider minifying for production at some point.

**Review by billyhunt — APPROVED** [review]
Looks really good.  Great work!

File: _assets/js/dist/accordion-compiled.js (hunk 1 above)
Comment (JackRyan1989) [reply _assets/js/dist/accordion-compiled.js:1]: Oh dang, thank you for catching this! It should have been minifying already. This shouldn't even have been tracked...

Comment (JackRyan1989) [reply _assets/js/dist/accordion-compiled.js]: @billyhunt The JS compiled assets should be removed now.

**Pull request merged**